packages/mongodb-memory-server-core/src/util/resolveConfig.ts

```diff
@@ -23,6 +23,7 @@ export enum ResolveConfigVariables {
   RUNTIME_DOWNLOAD = 'RUNTIME_DOWNLOAD',
   USE_HTTP = 'USE_HTTP',
   SYSTEM_BINARY_VERSION_CHECK = 'SYSTEM_BINARY_VERSION_CHECK',
+  USE_ARCHIVE_NAME_FOR_BINARY_NAME = 'USE_ARCHIVE_NAME_FOR_BINARY_NAME',
```

Comment:
from what i can tell, this option is currently missing from the docs, please add it to `docs/api/config-options` (in the style of the other options)